src/index.html

```diff
@@ -9,6 +9,21 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="header__logo"><img src="./images/logo.png" alt="logo"></a>
+      <nav class="header__menu">
+        <ul class="header__list">
+          <li class="header__item "><a href="#Apple" class="header__link is-active">Apple</a></li>
```

Comment:
```suggestion
          <li class="header__item ">
            <a href="#Apple" class="header__link is-active">
              Apple
            </a>
          </li>
```
same here, it's hard to read